pkg/sdk/go/things_test.go

```diff
@@ -1365,7 +1364,8 @@ func TestIdentify(t *testing.T) {
 func TestShareThing(t *testing.T) {
 	cRepo := new(mocks.Repository)
 	gRepo := new(gmocks.Repository)
-	uauth := cmocks.NewAuthService(users, map[string][]cmocks.SubjectSet{adminID: {uadminPolicy}})
+	aPolicy := cmocks.SubjectSet{Subject: "things", Relation: []string{"g_add", "c_share"}}
```

Comment:
Add an unhappy scenario where the token is OK, but the user is not allowed to share.